code/modules/surgery/organs/augments_chest.dm

```diff
@@ -213,6 +213,8 @@
 	switch(severity)
 		if(EMP_HEAVY)
 			owner.adjustFireLoss(35)
+			to_chat(owner, span_warning("Your thruster implant malfunctions and mildly burns you!"))
 		if(EMP_LIGHT)
 			owner.adjustFireLoss(10)
+			to_chat(owner, span_danger("Your thruster implant malfunctions and severely burns you!"))
```

Comment:
I think you have these reversed: you put "severely burns" under emp_light.

Reply:
Oop